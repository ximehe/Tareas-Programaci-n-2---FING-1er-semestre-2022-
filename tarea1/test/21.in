# implementar _rep_colCadenas, crearColCadenas, cantidadColCadenas, estaEnColCadenas
cantidadColCadenas 0
cantidadColCadenas 9
estaEnColCadenas 5 0
estaEnColCadenas 5 9
Fin