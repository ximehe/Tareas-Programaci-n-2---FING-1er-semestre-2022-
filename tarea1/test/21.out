1>#  implementar _rep_colCadenas, crearColCadenas, cantidadColCadenas, estaEnColCadenas.
2>La cantidad en la cadena 0 es 0.
3>La cantidad en la cadena 9 es 0.
4>5 no está en la cadena 0.
5>5 no está en la cadena 9.
6>Fin.
